### PR TITLE
Allow GTMSessionFetcher 3.x

### PR DESCRIPTION
**Pull request by petea**

### CODE REVIEW

**Review by thomasvl — APPROVED** [review]

**Review by thomasvl — APPROVED** [review]

**Review by paulb777 — APPROVED** [review]

**Review by mdmathias — APPROVED** [review]

**Review by Apple1D — COMMENTED** [review]
- #- [ ] [@](https://github.com/google/GTMAppAuth/pull/192/files)

File: Package.swift [hunk 1]
```diff
@@ -32,7 +32,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/google/gtm-session-fetcher.git", "1.5.0" ..< "3.0.0"),
+        .package(url: "https://github.com/google/gtm-session-fetcher.git", "1.5.0" ..< "4.0.0"),
         .package(url: "https://github.com/openid/AppAuth-iOS.git", "1.6.0" ..< "2.0.0")
```
Comment (Apple1D) [inline Package.swift:36]: _[image](https://github.com/google/GTMAppAuth/pull/192/files)_

Comment (Apple1D) [inline Package.swift:36]: Thank you

**Review by Apple1D — COMMENTED** [review]
- #- [ ] [@](https://github.com/google/GTMAppAuth/pull/192/files)

**Comment by Apple1D** [comment]
**_[https://user-images.githubusercontent.com/92694902/211060245-8be24820-2d3c-456d-ba42-a5d6033f6c68.MOV](https://github.com/google/GTMAppAuth/pull/192/files)_**

**Comment by Apple1D** [comment]
****

**Pull request merged**